CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Upgraded upstrem chart from 5.0.0 to 5.1.0 - see [changelog](https://github.com/grafana/loki/blob/main/production/helm/loki/CHANGELOG.md) for more information.
+- Upgraded Loki from 2.7.3 to 2.8.0 - see [changelog](https://github.com/grafana/loki/blob/main/CHANGELOG.md) for more information.
```

Comment:
Previous version was already running 2.8.0

Reply:
Removed this line and added another for loss of compability instead